specification/communication/data-plane/Identity/preview/2025-04-01-preview/CommunicationIdentity.json

```diff
@@ -311,6 +362,10 @@
     "CommunicationIdentityCreateRequest": {
       "type": "object",
       "properties": {
+        "externalId": {
```

Comment:
Should we add minLength 1 and maxLength 256?

Reply:
Done